plugin.video.vrt.nu/addon.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.vrtnu"
+       name="VRT Nu"
+       version="0.0.1"
+       provider-name="Martijn Moreel">
+
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+		<import addon="script.module.requests" version="2.12.4"/>
+		<import addon="script.module.beautifulsoup4" version="4.3.2"/>
+    </requires>
+
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>video</provides>
+    </extension>
+
+    <extension point="xbmc.addon.metadata">
+        <summary lang="nl">Met dit programma kan je alle streams van vrt.nu bekijken.</summary>
+        <description lang="nl">Met dit programma kan je alle streams van vrt.nu bekijken. Alsook de livestreams van één, Canvas en Ketnet!</description>
```

Comment:
please make it lang="nl_NL"  and add the mandatory English description and summary lang="en_GB".